Problem3.cpp

```diff
@@ -0,0 +1,38 @@
+// Time Complexity : O(N) N is size of string
+// Space Complexity : O(N)  
```

Comment:
Space will be constant here since you are using map which can at ax have all valid characters